### ISSUE TITLE
Not search in ldap

### ISSUE THREAD

**Issue by texnoid** [mfreiholz/iF.SVNAdmin#112]
Server1 (working)
CentOS 2.6.32-504.30.3.el6.x86_64
Package: 
PHP 5.3.3 (cli) (built: Jul  9 2015 17:39:00) 
Apache/2.2.15 (CentOS)
ldap:
LDAP Support	enabled
RCS Version	$Id: ldap.c 299434 2010-05-17 20:09:42Z pajoye $
Total Links	0/unlimited
API Version	3001
Vendor Name	OpenLDAP
Vendor Version	20439
SASL Support	Enabled

core prefork http_core mod_so mod_auth_basic mod_auth_digest mod_authn_file mod_authn_alias mod_authn_anon mod_authn_dbm mod_authn_default mod_authz_host mod_authz_user mod_authz_owner mod_authz_groupfile mod_authz_dbm mod_authz_default util_ldap mod_authnz_ldap mod_include mod_log_config mod_logio mod_env mod_ext_filter mod_mime_magic mod_expires mod_deflate mod_headers mod_usertrack mod_setenvif mod_mime mod_dav mod_status mod_autoindex mod_info mod_dav_fs mod_vhost_alias mod_negotiation mod_dir mod_actions mod_speling mod_userdir mod_alias mod_substitute mod_rewrite mod_proxy mod_proxy_balancer mod_proxy_ftp mod_proxy_http mod_proxy_ajp mod_proxy_connect mod_cache mod_suexec mod_disk_cache mod_cgi mod_version mod_php5 mod_dav_svn mod_authz_svn

iF.SVNAdmin:
1.6.3 UNOFFICIAL
Video: https://drive.google.com/file/d/0B1kn6CUg7swfeElBV0FPWHdMYmM/view?usp=drivesdk
=========================================
Server2 (not working)
Debian 3.16.7-ckt11-1+deb8u6 (2015-11-09) i686
PHP Version 5.6.14-0+deb8u1
Apache/2.4.10 (Debian)
LDAP:
LDAP Support	enabled
RCS Version	$Id: ab663b156be320c3bba8a7267187105adc47361a $
Total Links	0/unlimited
API Version	3001
Vendor Name	OpenLDAP
Vendor Version	20440
SASL Support	Enabled

core mod_so mod_watchdog http_core mod_log_config mod_logio mod_version mod_unixd mod_access_compat mod_alias mod_auth_basic mod_authn_core mod_authn_file mod_authz_core mod_authz_host mod_authz_svn mod_authz_user mod_autoindex mod_dav mod_dav_svn mod_deflate mod_dir mod_env mod_filter mod_mime prefork mod_negotiation mod_php5 mod_setenvif mod_status

iF.SVNAdmin:
1.6.3 UNOFFICIAL
 Video: https://drive.google.com/file/d/0B1kn6CUg7swfX0hHbGk2bGhTWVk/view?usp=drivesdk

**Comment by mfreiholz** [comment]
Hi,
it looks like both SVNAdmin environments access the same central LDAP server, right? So there shouldn't be a problem with the LDAP server itself.

Maybe it is a problem with PHP 5.6 and the newer LDAP library. Can't tell for sure right now. I need to setup a test-environment for further inspection.

Are there any related error logs in `/var/log/apache2/error.log`?

**Comment by texnoid** [comment]
Yes, it same central LDAP server. Perhaps the problem is in php or ldap module.
 */var/log/apache2/error.log* don't see critical problems or other

**Comment by texnoid** [comment]
Hey man, have you tried configuration php ldap ?

**Comment by mfreiholz** [comment]
Hey no sorry.
What kind of machine do you use for this server? I just saw that you used an i686 Debian Image.

**Comment by texnoid** [comment]
Yes, I used i686 debian image.
